addons/crm/models/crm_lead.py

```diff
@@ -1182,7 +1197,34 @@ def get_import_templates(self):
     # ------------------------------------------------------------
     # PLS
     # ------------------------------------------------------------
-
+    # Predictive lead scoring is computing the lead probability, based on won and lost leads from the past
+    # Each won/lost lead increments a frequency table, where we store, for each field/value couple, the number of
+    # won and lost leads.
+    #   E.g. : A won lead from Belgium will increase the won count of the frequency country_id='Belgium' by 1.
+    # The frequencies are split by team_id, so each team has his own frequencies environment. (Team A doesn't impact B)
+    # There are two main ways to build the frequency table:
+    #   - Live Increment: At each Won/lost, we increment directly the frequencies based on the lead values.
+    #       Done right BEFORE writing the lead as won or lost.
+    #       We consider a lead that will be marked as won or lost.
+    #       Used during usage, to ensure frequency table is always up to date
+    #   - One shot Rebuild: empty the frequency table and rebuild it from scratch, based on every already won/lost leads
+    #       Done AFTER the lead has been won or lost.
+    #       We consider all the leads that have been already won or lost.
+    #       Used in one shot, when modifying the criteria to take into account (fields or reference date)
+    #
+    # PLS methods will be split into multiple parts to ease comprehension,
+    #   - Probability Computation
+    #   - Live increment
+    #   - One shot Rebuild
+    #   - Tools/Common code
+    #       - Config Parameters
+    #       - Computation
+    #       - Frequency related methods
+    #       - Common
```

Comment:
Feels unnecessary.
If every sub-method has its own small docstring, it's good enough.